### PR TITLE
Documentation improvements to fix #6

### LINKED ISSUE #6: add a more complete example in the readme
I'd expect to have a full example of how to load tangram and use cartocss, like

```js

var cartocss = `
#layer {
   marker-fill: red;
   marker-width: 10;
}
`
scene_layer = Tangram.leafletLayer({
  scene: 'demos/scene.yaml',
  logLevel: 'debug'
}).addTo(map);

map.setView( [ 40.7, -74.009 ], 14 );
scene_layer.scene.config.layers.buildings.draw = CCSS.carto2Draw(cartocss);
scene_layer.scene.updateConfig();
});

```

Some other stuff:
- I'd also keep only one image (that it's related to the example)
- I'd publish the example live (using gh-pages) and add a link to it
- move the contributing section to a CONTRIBUTING file

### PR DESCRIPTION
**Pull request by donflopez**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Contributing
+
+```sh
+me$ git clone $repo_url
+cd into_folder
+yarn
```
Comment (javisantana) [inline CONTRIBUTING.md]: could you change by npm? everyone knows npm and people that know yarn already know that it's a npm replacement

Comment (donflopez) [inline CONTRIBUTING.md]: yep! sorry!!

**Comment by donflopez** [comment]
Done!

**Comment by javisantana** [comment]
:es:

**Pull request merged**